actions/docs-verifier/src/ActionRunner/Program.cs

```diff
@@ -20,9 +19,28 @@
     throw new InvalidOperationException($"The value of GITHUB_PR_NUMBER environment variable is not valid.");
 }
 
+static bool IsYmlOrMarkdownFile(PullRequestFile file) => Path.GetExtension(file.FileName) is ".yml" or ".md";
```

Comment:
Maybe add "yaml" for completeness?

```suggestion
static bool IsYmlOrMarkdownFile(PullRequestFile file) => Path.GetExtension(file.FileName) is ".yml" or ".yaml" or ".md";
```

Reply:
Now that I look at this again, a `ToLower` might be in order here as well (although we do standardize on lower case file names)

Reply:
@BillWagner I tried renaming a file from `yml` to `yaml` in a draft PR before, and open publishing build didn't generate a preview link for it. So they seem unsupported.